src/test/java/ca/uhn/fhir/jpa/starter/OperationCollectDataTest.java

```diff
@@ -142,10 +251,28 @@ private Parameters fetchParameter(String theUrl) throws IOException {
 		return parameters;
 	}
 
+	private Parameters fetchXmlParameter(String theUrl) throws IOException {
+		Parameters parameters;
+		HttpGet get = new HttpGet(theUrl);
```

Comment:
@gitcliff , you make unncesesary duplications , can you create the Get once in one private method and re-use that,with the existing tests ??